README.md

```diff
@@ -4,7 +4,7 @@ This plugin adds the [Loft CLI](https://github.com/loft-sh/loft) commands to [De
 
 You can add this plugin to devspace via:
 ```
-devspace add plugin https://github.com/loft-sh/devspace-plugin-loft --version=v3.4.6
+devspace add plugin https://github.com/loft-sh/devspace-plugin-loft --version=v4.0.0-alpha.11
```

Comment:
Should be correct. It is confusing since this plugin has always used the platform version, so it will still get updated when we do platform releases as part of that release process... but now it depends on the vcluster for the actual commands.. so the assets need to point to the vcluster CLI version that the platform uses.